pkg/storage/driver/secrets_test.go

```diff
@@ -184,3 +185,30 @@ func TestSecretUpdate(t *testing.T) {
 		t.Errorf("Expected status %s, got status %s", rel.Info.Status.String(), got.Info.Status.String())
 	}
 }
+
+func TestSecretDelete(t *testing.T) {
+	vers := 1
+	name := "smug-pigeon"
+	namespace := "default"
+	key := testKey(name, vers)
+	rel := releaseStub(name, vers, namespace, rspb.StatusDeployed)
+
+	secrets := newTestFixtureSecrets(t, []*rspb.Release{rel}...)
+
+	// perform the deletion on a non-existing release
+	_, err := secrets.Delete("nonexistent")
+	if err == nil || !strings.Contains(err.Error(), "not found") {
```

Comment:
This could be changed to the specific error.

```
if _, err := secrets.Delete("nonexistent"); err == ErrReleaseNotFound { ... }
```

Reply:
I've checked your suggestion. I didn't work as the "if" condition apierrors.IsNotFound(err) is False, so the code in the "if" block cannot be reached. The actual error came from the next line of code with value:
delete: failed to get release "nonexistent": release: not found